gradio/exceptions.py

```diff
@@ -73,12 +75,18 @@ def divide(numerator, denominator):
     Demos: calculator, blocks_chained_events
     """
 
-    def __init__(self, message: str = "Error raised."):
+    def __init__(
+        self,
+        message: str = "Error raised.",
+        duration: int | None = 10,
```

Comment:
nit but 
```suggestion
        duration: int | float | None = 10,
```

Reply:
just float is fine, int is a subset of float